src/docfx/validation/MetadataValidator.cs

```diff
@@ -76,7 +76,8 @@ public List<Error> ValidateMetadata(JObject metadata, FilePath filePath)
             foreach (var schemaValidator in _schemaValidators)
             {
                 // Only validate conceptual files
-                if (contentType == ContentType.Page && mime == "Conceptual" && !metadata.ContainsKey("layout"))
+                if (contentType == ContentType.Page && mime == "Conceptual" &&
+                    (!metadata.ContainsKey("layout") || string.Equals(metadata.Value<string>(), "conceptual", StringComparison.OrdinalIgnoreCase)))
```

Comment:
`Value<string>` will throw if the value is not a string?

Reply:
good catch!